core/config.lua

```diff
@@ -67,6 +67,12 @@ TC2.defaultConfig.bar = {
 	colorMod			= 0,									-- color modifier
 }
 
+-- self threat bar color settings
+TC2.defaultConfig.selfBar = {
+	enableCustomColor 		 = false,							-- enable custom color for player
```

Comment:
this can be changed to just `enabled` then.

Reply:
Renamed